ddtrace/internal/telemetry/logging.py

```diff
@@ -0,0 +1,67 @@
+import logging
+import os
+import traceback
+
+from ddtrace.internal.telemetry.constants import TELEMETRY_LOG_LEVEL
+
+
+class DDTelemetryLogHandler(logging.Handler):
+    CWD = os.getcwd()
+
+    def __init__(self, telemetry_writer):
+        self.telemetry_writer = telemetry_writer
+        super().__init__()
+
+    def emit(self, record):
+        # type: (logging.LogRecord) -> None
```

Comment:
nit: We should use python3 typing

```suggestion
    def emit(self, record: logging.LogRecord) -> None:
```